components/service/glean/src/main/java/mozilla/components/service/glean/private/LabeledMetricType.kt

```diff
@@ -213,4 +213,31 @@ data class LabeledMetricType<T>(
 
         return getMetricWithNewName(newMetricName)
     }
+
+    /**
+     * Get the specific metric for a given label.
+     *
+     * If a set of acceptable labels were specified in the metrics.yaml file,
+     * and the given label is not in the set, it will be recorded under the
+     * special [OTHER_LABEL].
+     *
+     * If a set of acceptable labels was not specified in the metrics.yaml file,
+     * only the first 16 unique labels will be used. After that, any additional
```

Comment:
used? recorded? So what happens if no labels are specified in the definition but someone tries to get by label index? Should that be an error?

Reply:
Ouch, sorry, that was some hasty copy 🍝 . I've fixed this. We only support indices for static labels.